src/set_theory/cardinal.lean

```diff
@@ -212,6 +212,11 @@ by rw [_root_.mul_comm b c];
 from (quotient.induction_on₃ a b c $ assume α β γ,
   quotient.sound ⟨equiv.arrow_arrow_equiv_prod_arrow γ β α⟩)
 
+lemma monoid_pow_eq_cardinal_pow {κ : cardinal.{u}} :
```

Comment:
This doesn't seem like the right name. Maybe `pow_cast_right` and swap the equality and make it `simp`?

Reply:
In fact, it wasn't the right lemma -- `^` was interpreted as a different pow in this setting. Fixed.